### PR TITLE
[feature/calens-changelog-actions] Automated Calens Changelog Creation

### PR DESCRIPTION
**Pull request by hosy**
## Description
This PR uses GitHub Actions to automatically generate a changelog file with Calens and commits the new CHANGELOG.md into the current branch.
Furthermore the PR template was updated with new checkboxes to get sure, that a release notes issue file for Calense was added.

## Motivation and Context
Automatically changelog file creation, with the same structure, as other ownCloud repositories.

## How Has This Been Tested?
- Add an release notes issue file into `changelog/unreleased` with the issue number as file name
- Commit and push to git
- Check GitHub Actions
- As a result the `CHANGELOG.md` file should be updated

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [x] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] I have read the [**CONTRIBUTING**](https://github.com/owncloud/ios-app/blob/master/CONTRIBUTING.md) document.
- [ ] I have added tests to cover my changes.
- [ ] All new and existing tests passed.

### CODE REVIEW

**Review by TheOneRing — COMMENTED** [review]
changelog/CHANGELOG.md
I guess you copied it from desktop, that file was committed by accident 🙈

File: .github/workflows/calens.yml [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+on:
+  push:
+    branches:
+    - fix/*
+    - feature/*
+    - milestone/*
+
+jobs:
+  build:
+    runs-on: ubuntu-16.04
```
[the remark this answers is not in the document]
Comment (hosy) [reply .github/workflows/calens.yml:12]: What do you prefer? 20.04?

**Comment by hosy** [comment]
> changelog/CHANGELOG.md
> I guess you copied it from desktop, that file was committed by accident 🙈

I used the file and folder structure from the `client` repo.

**Review by TheOneRing — APPROVED** [review]
LGTM

**Review by jesmrec — APPROVED** [review]

**Review by micbar — APPROVED** [review]
Looking god so far.

If you would like to take a look in ocis, we managed to do release and Full changelog with only a single template.

**Comment by micbar** [comment]
Do you use Calens in the release process?

We do that in ocis like this https://github.com/owncloud/ocis/blob/master/.drone.star#L914

**Comment by hosy** [comment]
@micbar thank you for your hint. Calens will be processed after each commit in the related branch.
I can have a look creating a GitHub release automatically, when a new tag was detected.

**Pull request merged**